projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py

```diff
@@ -445,6 +445,9 @@ def aten〇std〇correction〡shape(self: List[int], dim: Optional[List[int]] =
 def aten〇argmax〡shape(self: List[int], dim: Optional[int] = None, keepdim: bool = False) -> List[int]:
     return upstream_shape_functions.argmax(self, dim, keepdim)
 
+def aten〇argmin〡shape(self: List[int], dim: Optional[int] = None, keepdim: bool = False) -> List[int]:
+    return upstream_shape_functions.argmax(self, dim, keepdim)
```

Comment:
Is there an `argmin` shape function? I assume it would do the same thing but would be a little less strange to look at :)

Reply:
There is none. see [here](https://github.com/pytorch/pytorch/blob/fcf6a76108be8e7b6db528b631a7b8ebdc7470ac/torch/jit/_shape_functions.py#L1068). I had the same issue with the aten.min.dim operator recently. I decided that it is not worth the effort to add it to pytorch, but perhaps I was wrong. @frafranz  Perhaps add a comment here to explain that using the argmax shape function for argmin is intentional?

Reply:
That is a good idea, I will add that comment!

Reply:
Comment is fine, definitely don't go to the effort of adding to pytorch. I just figured it was worth asking, thanks for the attention to detail!